src/sentry/tasks/auth/__init__.py

```diff
@@ -0,0 +1,4 @@
+from sentry.taskworker.registry import TaskNamespace, taskregistry
```

Comment:
We could move this file and the auth tasks to `sentry.auth.tasks` (can be done separately). That would avoid the duplicate segment in the import path.